### PR TITLE
[WIP] TRD Trap Simulator

### PR DESCRIPTION
**Pull request by bazinski**
TRD Trapsimulator.

### CODE REVIEW

**Comment by bazinski** [comment]
@martenole let me fix the above stuff, before you waste your time away from your writing.

File: Detectors/TRD/base/include/TRDBase/Digit.h [hunk 1]
```diff
@@ -46,11 +46,13 @@ class Digit : TimeStamp
   void setRow(int row) { mRow = row; }
   void setPad(int pad) { mPad = pad; }
   void setADC(ArrayADC const& adc) { mADC = adc; }
+  void setTime(double eventtime) { mTime = eventtime; }
```
Comment (jolopezl) [inline Detectors/TRD/base/include/TRDBase/Digit.h]: You should rebase to dev your PR. The current Digit class inherits TimeStamp from the common dataformats.

Comment (jolopezl) [reply Detectors/TRD/base/include/TRDBase/Digit.h]: So, you will have to rebase to dev after this PR https://github.com/AliceO2Group/AliceO2/pull/2840.

Comment (bazinski) [reply Detectors/TRD/base/include/TRDBase/Digit.h]: @jolopezl  Yes

**Pull request closed without merging**